src/components/todoList/TodoList.js

```diff
@@ -0,0 +1,121 @@
+import { Component } from 'react';
+import './todoList.scss';
+
+export class TodoList extends Component {
+  state = {
+    todos: [],
+    initialTodos: [],
+    filterValue: ''
+  }
+
+  componentDidMount() {
+    fetch('https://jsonplaceholder.typicode.com/todos')
+      .then(response => response.json())
+      .then(todos => this.setState({
+        todos: todos.slice(0, 20),
+        initialTodos: todos.slice(0, 20)
+      }));
+  }
+
+  deleteTodo = (id) => {
+    const { todos } = this.state;
+    this.setState({
+      todos: todos.filter(todo => todo.id !== id),
+      initialTodos: todos.filter(todo => todo.id !== id)
+    });
+  }
+
+  completeTodo = (todoId) => {
+    const { todos } = this.state;
```

Comment:
actually, you shouldn't implement it (all todo manipulations will be made via server), but ok, leave it as it is